### PR TITLE
Rename ambiguous variable in Punishments monitor

### PR DESCRIPTION
**Pull request by panpawn**

### CODE REVIEW

**Comment by panpawn** [comment]
>varliable

File: punishments.js [hunk 1]
```diff
@@ -1032,12 +1032,12 @@ Punishments.getRoomPunishments = function (user, publicOnly) {
  * @param {User} user
  */
 Punishments.monitorRoomPunishments = function (user) {
-	let num = (Config.monitorminpunishments !== undefined ? Config.monitorminpunishments : 3); // Default to 3 if the config option isn't present.
-	if (!num) return;
+	const MINIMUM_PUNISHMENTS_TRIGGER = (typeof Config.monitorminpunishments === 'number' ? Config.monitorminpunishments : 3); // Default to 3 if the Config option is not defined or valid
```
Comment (Zarel) [inline punishments.js]: We do not use allcaps/underscores for constants unless they're actually constants, not config values.

File: punishments.js [hunk 2]
```diff
@@ -1032,12 +1032,12 @@ Punishments.getRoomPunishments = function (user, publicOnly) {
  * @param {User} user
  */
 Punishments.monitorRoomPunishments = function (user) {
-	let num = (Config.monitorminpunishments !== undefined ? Config.monitorminpunishments : 3); // Default to 3 if the config option isn't present.
-	if (!num) return;
+	const minimumMonitorTrigger = (typeof Config.monitorminpunishments === 'number' ? Config.monitorminpunishments : 3); // Default to 3 if the Config option is not defined or valid
```
Comment (panpawn) [inline punishments.js]: >_>

**Comment by Zarel** [comment]
After some discussion, I think `minPunishments` would probably be best here.

File: punishments.js (hunk 2 above)
Comment (Zarel) [inline punishments.js]: use `\` to escape

\>_>

**Comment by Zarel** [comment]
(As the variable name)

**Comment by panpawn** [comment]
ok

File: punishments.js [hunk 3]
```diff
@@ -1050,4 +1050,4 @@ Punishments.monitorRoomPunishments = function (user) {
 
 		Monitor.log(`[PunishmentMonitor] ${user.name} currently has punishments in ${punishments.length} rooms: ${punishmentText}`);
 	}
-};
\ No newline at end of file
+};
```
Comment (Zarel) [inline punishments.js]: @bumbadadabum, this actually _is_ a problem though; be more careful about trailing whitespace...

**Pull request merged**